src/content/en/tools/lighthouse/audits/budgets.md

```diff
@@ -0,0 +1,283 @@
+
+<figure>
+  <img src="images/budgetscustom.png"
+       alt="A custom budget report."/>
+  <figcaption>
+    <b>Figure X</b>. A custom budget report.
+  </figcaption>
+</figure>
+
+When you define a budget file, the **Performance Budgets** audit reports on the resource types that you've specified.
+But you can still access the information for the rest of the types from the **Keep Request Counts And File Sizes Small**
+audit.
+
+Example JSON output:
+
+```json
+"resource-budget": {
+  "id": "resource-budget",
+  "title": "Keep request counts and file sizes small",
+  "description": "To set budgets for the quantity and size of page resources, add a budget.json file.",
+  "score": null,
+  "scoreDisplayMode": "informative",
+  "details": {
+    "type": "table",
+    "headings": [
+      {
+        "key": "label",
+        "itemType": "text",
+        "text": "Resource Type"
+      },
+      {
+        "key": "count",
+        "itemType": "numeric",
+        "text": "Requests"
+      },
+      {
+        "key": "size",
+        "itemType": "bytes",
+        "text": "File Size"
+      },
+      {
+        "key": "countOverBudget",
+        "itemType": "text",
+        "text": ""
+      },
+      {
+        "key": "sizeOverBudget",
+        "itemType": "bytes",
+        "text": "Over Budget"
+      }
+    ],
+    "items": [
+      {
+        "label": "Script",
+        "count": 2,
+        "size": 444321,
+        "sizeOverBudget": 316321
+      },
+      {
+        "label": "Total",
+        "count": 21,
+        "size": 650255,
+        "sizeOverBudget": 138255
+      },
+      {
+        "label": "Image",
+        "count": 6,
+        "size": 80392
+      },
+      {
+        "label": "Third-party",
+        "count": 13,
+        "size": 112138,
+        "countOverBudget": "13 requests"
+      }
+    ]
+  }
+}
+```
+
+See the following pages to learn more about performance budgets:
+
+* [Start Performance Budgeting](https://addyosmani.com/blog/performance-budgets/){: .external }
+* [Performance budgets 101](https://web.dev/performance-budgets-101/){: .external }
+* [Your first performance budget](https://web.dev/your-first-performance-budget/){: .external }
+* [Incorporate performance budgets into your build process](https://web.dev/incorporate-performance-budgets-into-your-build-tools/){: .external }
+
+## Setup {: #setup }
+
+Follow the instructions below to configure Lighthouse to report when you're over budget. As mentioned
+before, this is only possible when running Lighthouse from the command line. The **Keep Request Counts
+And File Sizes Small** audit is always available in all Lighthouse runtime environments and requires no configuration.
+
+### Define a budget file {: #define }
+
+By convention the budget file is usually called `budget.json` but you can call it whatever you like.
+The example below sets the following budgets:
+
+* 125 kilobytes for all scripts
```

Comment:
Maybe reword slightly to indicate that the budget applies to the scripts (& stylesheets) collectively, rather than individually

Reply:
Done